pdbufr/bufr_structure.py

```diff
@@ -135,7 +136,58 @@ def wmo_station_id_from_bufr(observation, prefix, keys):
     block_number = int(observation[prefix + keys[0]])
     station_number = int(observation[prefix + keys[1]])
     return block_number * 1000 + station_number
-
+    
+def wmo_station_position_from_bufr(observation, prefix, keys):
+    # type: (T.Dict[str, T.Any], str, T.List[str]) -> Point
+    longitude = float(observation[prefix + keys[0]]) # easting (X)
+    latitude = float(observation[prefix + keys[1]]) # northing (Y)
+    heightOfStationGroundAboveMeanSeaLevel = float(observation.get(prefix + keys[2], 0.0))
+    return Point([longitude,latitude,heightOfStationGroundAboveMeanSeaLevel]) 
+
+def CRS_from_bufr(observation, prefix, keys):
```

Comment:
I would prefer this to be a dictionary instead of a long `if... elif`

Reply:
Very good hint - included